src/components/VTextField/VTextField.spec.js

```diff
@@ -178,7 +177,6 @@ test('VTextField.js', ({ mount }) => {
     icon.trigger('click')
     await wrapper.vm.$nextTick()
     expect(wrapper.vm.inputValue).toBe('foo')
-    expect(called).toBe(0)
```

Comment:
You can use `jest.fn()` to mock this (in the previous test)

```js
const callback = jest.fn()
...
appendIconCb: callback
...
expect(callback.mock.calls.length).toBe(1)

Reply:
done